mmdet3d/ops/roiaware_pool3d/points_in_boxes.py

```diff
@@ -17,6 +17,7 @@ def points_in_boxes_gpu(points, boxes):
     """
     assert boxes.shape[0] == points.shape[0]
     assert boxes.shape[2] == 7
+    assert points.shape[2] == 3
```

Comment:
, f'got unexpected shape {points.shape[2]}'

Reply:
OK, I will modify it